### ISSUE TITLE
Build a robust gui using tkinter that can handle almost every desktop organizing task initiated in the backend

### ISSUE THREAD

**Issue by sesiii** [Ash515/AutomaDesk#23]
Using Tkinter to build a interface whuc is simple to use and also versatile to use on any operating system

**Comment by mvp-2003** [comment]
Yes this is what I am talking about.
A GUI app that starts a thread that runs in the background to do the functions.

**Comment by mvp-2003** [comment]
@sesiii I would recommend CustomTkinter, which is a more advanced and upgraded version of Python Tkinter library. CustomTkinter creates more interesting visuals and eye-catching.

**Comment by karanmanglani** [comment]
I can work on this issue since i have worked on tkinter previously for many projects.

Please assign me this issue indee gssoc 24 and give it a level also

**Comment by mvp-2003** [comment]
@sesiii this would need some class and objects so that different automation tabs open up their respective pages on which their interactable are provided.

**Comment by Santana-aa** [comment]
I have done few projects using Python's Tkinter library and I want to contribute to this project. Please assign me this issue. 
Projects that I have done using Tkinter are-

1.  [](https://github.com/Santana-aa/weather_app)
2.  [](https://github.com/Santana-aa/randomPasswordGenerator)

**Comment by mvp-2003** [comment]
@sesiii we need to enhance the UI using CustomTkinter. If you are ready for that, we both can work on it together. If you want to join in, we already have a few people trying to work on the UI/UX. We need to come up with a common UI.

**Comment by sesiii** [comment]
Sure, I am in 
Will work together then

**Comment by sesiii** [comment]
What is the update on this @mvp-2003?
are you working on this?

**Comment by mvp-2003** [comment]
Yes I am using CustomTkinter. I have created the main frames and 2 sub-pages. I will upload it soon. You can take from there, because I will have to get started with the multi-threading and system calls part.
Also I am waiting a little for the personal project channels to get created in Discord. Communication is tough.
